### PR TITLE
#204 Enabling CSRF protection

### PR DESCRIPTION
**Pull request by bkuzmic**
For #204:

- Enabling CSRF protection in application
- Implementing for table shapes's `create_edit` template
- Leaving puzzle to continue implementation for other templates

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#351](https://github.com/timelesslounge/timelessis/issues/351) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#351](https://github.com/timelesslounge/timelessis/issues/351) is assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/97cb47b0-5572-4195-acb9-0ce7eb1e7d52); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/restaurants/table_shapes/views.py [hunk 1]
```diff
@@ -39,14 +39,9 @@ def create():
 
     if request.method == "POST" and form.validate():
         """
-        @todo #162:30min Initialize CSRF token protection for app and add
-         token to this template. Implementation detail you can find by the
-         following page: https://flask-wtf.readthedocs.io/en/stable/csrf.html
         @todo #162:30min This form currenly cannot save pictures. Need have a
```
Comment (ammaratef45) [inline timeless/restaurants/table_shapes/views.py]: @bkuzmic PDD complained about this
>puzzle at line #42; Space expected at 45:8; make sure all lines in the puzzle body have a single leading space.

**Comment by ammaratef45** [comment]
@bkuzmic there is a problem with a puzzle. please fix it.

**Comment by bkuzmic** [comment]
@ammaratef45 Thanks, it should be fixed now.

**Comment by ammaratef45** [comment]
@bkuzmic Great :+1:

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 1 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#351](https://github.com/timelesslounge/timelessis/issues/351) is not in the agenda of @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), can't inspect

**Comment by 0crat** [comment]
The job [#351](https://github.com/timelesslounge/timelessis/issues/351) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ammaratef45** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/351#issuecomment-465935298))

@ammaratef45 This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#351](https://github.com/timelesslounge/timelessis/issues/351) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 8min
  * @ammaratef45[/z](https://www.0crat.com/u/ammaratef45) will receive 20 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23351%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ammaratef45** [comment]
@ypshenychka zerocrat removed this from my agenda again, quality review needed here :laughing:

**Comment by ypshenychka** [comment]
@ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ammaratef45** [comment]
> @ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
> 
> > The code reviewer found at least three problems in the code.
> > Comments were mostly about design problems, not cosmetic issues.
> 
> Only one issue was found during code review.
> Please confirm that you'll try to find at least three major problems while future reviews.

@ypshenychka confirm

**Comment by ypshenychka** [comment]
@ammaratef45 thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**